src/pkg/packager/deploy.go

```diff
@@ -453,10 +455,11 @@ func (p *Packager) setupState(ctx context.Context) (err error) {
 	defer spinner.Stop()
 
 	state, err := p.cluster.LoadZarfState(ctx)
-	// Return on error if we are not in YOLO mode
+	// We ignore the error if in YOLO mode because Zarf should not be initiated.
 	if err != nil && !p.cfg.Pkg.Metadata.YOLO {
```

Comment:
IMO we should only ignore the error in yolo mode if the error is `kerrors.IsNotFound(err)`